src/ensembl/production/metadata/api/adaptors/genome.py

```diff
@@ -520,7 +521,7 @@ def fetch_sequences(self, genome_id=None, genome_uuid=None, assembly_uuid=None,
         # These options are in order of decreasing specificity,
         # and thus the ones later in the list can be redundant.
         if genome_id is not None:
-            seq_select = seq_select.filter(Genome.genome_id == genome_id)
+            seq_select = seq_select.filter(Genome.genome_id.in_(genome_id))
```

Comment:
What is the type expected here for genome_id ? Is it string or list or anything else ?  The doc string says it is expected as int.

Reply:
It's a list of `int` (`check_parameter()` in line 504 parses into a list if it's not)